packages/web/src/components/spa/HackAUBG/GradingCriteria/GradingCriteria.jsx

```diff
@@ -0,0 +1,13 @@
+import { GradingCriteriaDesktop } from './GradingCriteriaDesktop/GradingCriteriaDesktop';
+import { GradingCriteriaMobile } from './GradingCriteriaMobile/GradingCriteriaMobile';
+import { useMediaQuery } from 'react-responsive';
+
+export const GradingCriteria = () => {
+    const isMobile = useMediaQuery({ query: '(max-width: 900px)' });
```

Comment:
We usually use `1000px` as the breaking point between mobile and desktop devices. Can you test if your implementation would work with 1000px instead of 900?